apps/apps.txt

```diff
@@ -13,3 +13,4 @@ Git, VCS, debian, false, true, false, 1
 LibreOffice, Office, debian, false, true, false, 1
 Gimp, Office, debian, false, true, false, 1
 Inkscape, Office, debian, false, true, false, 1
+Idle-3.5, office, debian, false, true, false, 1
```

Comment:
Can you drop the `-3.5` and fix the case on `office` to be `Office`?

Reply:
sure